src/Http/src/Request/FilesBag.php

```diff
@@ -13,7 +13,6 @@
  * @method UploadedFileInterface|null get(int|string $name, $default = null)
  * @method UploadedFileInterface[] all()
  * @method UploadedFileInterface[] fetch(array $keys, bool $fill = false, $filler = null)
- * @method \Traversable<array-key, UploadedFileInterface> getIterator()
```

Comment:
Need to replace with `@inplements IteratorAggregate<...>`

Reply:
Added @inplements IteratorAggregate